### ISSUE TITLE
How to Auto-like my own feed?

### ISSUE THREAD

**Issue by dizawish** [LevPasha/instabot.py#361]
How to Auto-like my own feed?
Is it possible?
Thanks in advance

**Comment by Killerherts** [comment]
https://github.com/virginsince1997/Photoliker

**Comment by Killerherts** [comment]
https://github.com/chrisjluc/Photoliker

**Comment by dizawish** [comment]
Hi, thank you for the support.
I downloaded Phantom.js and ran succesfully some of the examples they provide.
However i downloaded and ran Photoliker and i get the following in the cmd:

ReferenceError: Can't find variable: $

  undefined:3
  :9
TypeError: null is not an object (evaluating 'point.x')

  phantomjs://code/Photoliker.js:58 in clickElement
  phantomjs://code/Photoliker.js:76
  phantomjs://code/Photoliker.js:167

Found nothing in Google or any hint in Phantom.js documentation.
